lib/Utils/ReplaceIRVar.cpp

```diff
@@ -41,17 +41,28 @@ void ReplaceIRVar::init(llvm::Module &M) {
   }
 }
 
+/// insert IR Name and Source Name mapping
+void ReplaceIRVar::insert(std::string Name){
```

Comment:
Would you like to create a separate pull request for the changes related to replace IR vars?

Reply:
Yes I will.